.eslintrc.js

```diff
@@ -11,36 +11,42 @@ module.exports = {
     browser: true,
     node: true,
   },
-  extends: ['plugin:import/recommended', 'airbnb', 'prettier', 'prettier/react'],
-  parser: 'babel-eslint',
+  extends: [
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'airbnb-typescript',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 7,
-    sourceType: 'module',
+    // project: './tsconfig.json',
   },
-  plugins: ['babel', 'material-ui', 'react-hooks'],
+  plugins: ['material-ui', 'react-hooks', '@typescript-eslint'],
   settings: {
     'import/resolver': {
       webpack: {
         config: path.join(__dirname, './docs/webpackBaseConfig.js'),
       },
     },
   },
-  /**
-   * Sorted alphanumerically within each group. built-in and each plugin form
-   * their own groups.
-   */
+  // Sorted alphanumerically
   rules: {
+    '@typescript-eslint/dot-notation': 'off', // Too slow
```

Comment:
Why?

Reply:
If you enable the rules that are disabled with `// Too slow`, you will time out the CI. No answer after 10 minutes. It seems to get stuck. I haven't looked at why.